docs/movement-node-experimental/l-monninger/open-network/README.md

```diff
@@ -0,0 +1,60 @@
+# `l-monninger/open-network`
```

Comment:
Is this meant to be the final name, or should there be a long-running branch named something like `open-network`?

Reply:
This is a long-running branch intended for external collaboration.

Reply:
If it gets too verbose in implementation though, we'll cut it.

Reply:
I can drop my name from it if you like haha